lib/engine/ability/hexes_bonus.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Ability
+    class HexesBonus < Base
```

Comment:
HexesBonus kind of reads funny, slight preference to just calling this HexBonus with hexes reader

Reply:
Done